source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-user.tsx

```diff
@@ -107,6 +110,12 @@ export function withCurrentUser() {
           // https://github.com/json-api-dotnet/JsonApiDotNetCore/issues/39
           // is resolved
           const response = await authenticatedGet('/api/users/current-user?include=organization-memberships,group-memberships');
+
+          if (this.isUserLocked(response)) {
+            console.debug('Current user is Locked');
+            throw new Error('Your user has been locked, contact you organization administrator');
```

Comment:
why are you throwing an error?

Reply:
I'm throwing an error so I can reuse the error page

        if (error) {
          return <PageError error={error} />;
        }

Reply:
:+1: